frontend/src/scenes/data-warehouse/external/forms/SourceForm.tsx

```diff
@@ -19,6 +19,7 @@ const sourceFieldToElement = (field: SourceFieldConfig, sourceConfig: SourceConf
             <LemonField key={field.name} name={[field.name, 'enabled']} label={field.label}>
                 {({ value, onChange }) => (
                     <>
+                        {!!field.caption && <p>{field.caption}</p>}
```

Comment:
just checking if this is intended or if should be `{field.caption && <p>{field.caption}</p>}` ?

Reply:
I did intentionally double-negate it - the results are the same though, right?

Reply:
ah right, sorry, yes you're right. I'm just not used to seeing it that often which is why I got confused. Ignore me!